### PR TITLE
Exercise 2

### PR DESCRIPTION
**Pull request by FrancoisLeclerc**
Add a normalize.css file in the css directory + fix indent in main.css

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    }
+    nav li {
+        display: inline;
+        border: 1px solid black;
+        padding: 5px;
+        margin-left: 10px;
+    }
+    nav a {
+        text-decoration: none;
+    }
```
Comment (nyanofthemoon) [inline css/main.css]: @FrancoisLeclerc Please fix identation.

File: index.html [hunk 2]
```diff
@@ -3,44 +3,10 @@
...
-                display: inline;
-                border: 1px solid black;
-                padding: 5px;
-                margin-left: 10px;
-            }
-            nav a {
-                text-decoration: none;
-            }
-        </style>
+         <link rel="stylesheet" href="css/normalize.css">
```
Comment (nyanofthemoon) [inline index.html]: @FrancoisLeclerc Please fix identation.

**Comment by FrancoisLeclerc** [comment]
done!

**Comment by ziad-saab** [comment]
@FrancoisLeclerc maybe you forgot to push your commits? Nothing has changed since the last comment

**Comment by ziad-saab** [comment]
@FrancoisLeclerc thanks! Next time please put a comment after your commit saying @ with the username of the person who is waiting for you.

**Pull request closed without merging**